kernel/core/plotting/plotting.cpp

```diff
@@ -3675,9 +3675,12 @@ std::vector<std::string> Plot::separateFunctionsAndData()
         sToken = getNextArgument(sFuncTemp, true);
         size_t nPos = sToken.find_first_of("#\"");
 
+        if (sToken.find_first_of("([{") < nPos)
+            nPos = sToken.find_first_of("#\"", getMatchingParenthesis(sToken));
+
         // Ensure we don't have a string expression right here
-        if (!nPos || NumeReKernel::getInstance()->getStringParser().isStringExpression(sToken.substr(0, nPos)))
-            throw SyntaxError(SyntaxError::CANNOT_PLOT_STRINGS, sCurrentExpr, sToken, sToken);
+        //if (!nPos || NumeReKernel::getInstance()->getStringParser().isStringExpression(sToken.substr(0, nPos)))
+        //    throw SyntaxError(SyntaxError::CANNOT_PLOT_STRINGS, sCurrentExpr, sToken, sToken);

```

Comment:
Should this be commented out or deleted entirely if we intend to allow column labels for headers and therefore plotting from now on?

Reply:
Is needed for the experimental feature, yes. Can be left for now, might be needed in the future.